index.html

```diff
@@ -118,6 +155,7 @@ <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
             <button type="button" > See Project</button>
         </div> 
     </div>
+    </div>
 </section>
 <section class="about-me">
     <div class="container-aboutme">
```

Comment:
- [ ] please can you work on centering the contents of your about me section, as shown from the image below it is not properly centered. It is important to always follow design structure as this shows how much attention to details you pay as a developer and it is an awesome quality to have :+1: 
- [ ] Also the third card has overflowing contents please kindly fix this as well :+1: 
- your design
![not-c-s](https://user-images.githubusercontent.com/89055502/236342263-31541baf-74fb-4832-bf07-7c8203beb561.png)